src/App.js

```diff
@@ -41,7 +41,7 @@ export default function Todos() {
           </button>
         </form>
         {toDoList?.length === 0 && <p className="text-2xl">No Todos</p>}
```

Comment:
You don't need an Elvis operator here, as your default value is an array, which means toList will always have the length property in this case.
```suggestion
        {toDoList.length === 0 && <p className="text-2xl">No Todos</p>}
```